### PR TITLE
Remove unnecessary imports in nav bar html.

### PR DESCRIPTION
**Pull request by j0n3lson**
The site navigation works without these imports and having them
complicates the webpackification process.

Related to #99

### CODE REVIEW

**Comment by j0n3lson** [comment]
Hi  @j0n3lson, 

 A random law from [Hacker Laws repository](https://github.com/dwmkerr/hacker-laws) is a small gift to thank you for opening the PR.

-----------------------------------------------------
Conway's Law
-----------------------------------------------------

Conway's Law on Wikipedia

This law suggests that the technical boundaries of a system will reflect the structure of the organisation. It is commonly referred to when looking at organisation improvements, Conway's Law suggests that if an organisation is structured into many small, disconnected units, the software it produces will be. If an organisation is built more around 'verticals' which are orientated around features or services, the software systems will also reflect this.

See also:

- The Spotify Model

-----------------------------------------------------
         github.com/dwmkerr/hacker-laws by Dave Kerr 
-----------------------------------------------------

**Comment by Steve-Leke** [comment]
Hey!

We actually do need a lot of these imports. I'm not sure if you checked the preview link, but none of the navbars work on this preview window. Index.html and Demo.html aren't using the custom tags. That's why it may seem as if the imports are being used, but they actually are.

File: public/templates/navBar.html [hunk 1]
```diff
@@ -4,54 +4,15 @@
...
 <header class="header-area header-sticky">
   <div class="container">
     <div class="row">
       <div class="col-12">
         <nav class="main-nav">
           <!-- ***** Logo Start ***** -->
-          <a href="/templates/index.html" class="logo scroll-to-section">Melly</a>
+          <a href="/templates/index.html" class="logo scroll-to-section"
+            >Melly</a
+          >
```
Comment (lillianchou94) [inline public/templates/navBar.html:22]: can we fix this plsss

Comment (lillianchou94) [reply public/templates/navBar.html:22]: fix as in format heh

**Review by lillianchou94 — DISMISSED** [review]

File: public/templates/navBar.html (hunk 1 above)
Comment (j0n3lson) [reply public/templates/navBar.html:22]: Hey so this is being auto formatted (probably because the line length)...

**Review by Steve-Leke — APPROVED** [review]

**Comment by Steve-Leke** [comment]
When we get the basic site to work, we should use import js libraries like imgix.js and bootstrap.js, to make the website more responsive. At the moment this site doesn't work on devices with smaller screens. Using those libraries could help fix that!

**Pull request merged**